docusaurus/docs/reference/ha/topology.md

```diff
@@ -0,0 +1,81 @@
+---
+sidebar_label: Topology
+sidebar_position: 60
+---
+
+# Controller Topology
+
+This document discuss considerations for how many controllers a network might 
+need and how to place them geographically.
+
+## Number of Controllers
+
+### Management
+
+The first consideration is how many controllers the network should be able to lose without losing
+functionality. A cluster of size N needs (N/2) + 1 controllers active and connected to be able
+to take model updates, such as provisioning identities, adding/changes services and updating policies.
+
+Since a two node cluster will lose some functionality if either node becomes unavailable, a minimum
+of 3 nodes is recommended.
+
+### Clients
+
+The functionality that controllers provide to clients doesn't require any specific number of controllers.
+A network manager will want to scale the number controllers based on client demand and may want to 
+place additional controllers geographically close to clusters of clients for better performance.
```

Comment:
A latency expressed as an order of magnitude in milliseconds (<1, 10, 100) would be helpful for understanding what's meant by "close." Etcd informs the reader that updates can only be accomplished as quickly as network and disk IO allow, emphasizing the significance of network had disk IO latency.

Reply:
Added some examples using latency in millis